pkg/dashboard/assets/js/api-token.js

```diff
@@ -50,9 +50,19 @@
 
   submitBtn.addEventListener("click", function (e) {
     e.preventDefault();
+
     if (apiTokenInput.validity.valid) {
-      window.location.reload();
-      localStorage.setItem("apiKey", apiTokenInput.value);
+      const inputApiToken = apiTokenInput.value.trim();
+      fetch(
+        `${window.INSIGHTS_HOST}/v0/oss/instance-types?ossToken=${inputApiToken}`
+      ).then((response) => {
+        if (response && response.status !== 500) {
+          window.location.reload();
+          localStorage.setItem("apiKey", apiTokenInput.value.trim());
+        } else {
+          alert("Invalid API Token");
```

Comment:
Do you want to do the new alert style in this PR, or a future one?

<img width="778" alt="email" src="https://user-images.githubusercontent.com/7611973/187688563-1ae27757-2d30-4026-a1cc-35887f03a4f8.png">

Reply:
I updated the code. Thank you